WTWSMS/decisions/WtWSMS_realm_decisions.txt

```diff
@@ -350,6 +350,10 @@ decisions = {
 		effect = {
 			set_global_flag = rome_restored
 			narrative_event = { id = ancrel.2220 tooltip = EVTTOOLTIPLoR.20 }
+			add_law = succ_primogeniture
+			add_law = vice_royalty_2
+			add_law = imperial_administration
+			set_allow_free_vice_royalty_revokation = yes
```

Comment:
I don't want those who restore the roman empire to receive a free boost in terns of laws.